pkgs/tools/security/aws-mfa/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "aws-mfa";
+  version = "0.0.12";
+  format = "wheel";
+
+  src = python3Packages.fetchPypi {
+    inherit version format;
+    pname  = "aws_mfa";
+    sha256 = "aef772e38734f1a50e618a2b19a8a69e5f0ced9cbeaf3be5132e9e1ba04fa294";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ boto3 ];
+
+  meta = {
+    description = "Easily manage your AWS Security Credentials when using Multi-Factor Authentication";
+    homepage = "https://github.com/broamski/aws-mfa";
+    license = with lib.licenses; [ mit ];
```

Comment:
```suggestion
    license = with licenses; [ mit ];
```

Reply:
I think I tried this and it would blow up with some errors about functions not having arguments or some such.

I'm not really good with the language to be honest.

Reply:
you would also need to apply my other suggestion

```
-- meta = {
++ meta = with lib; {
```